releasing.md

```diff
@@ -8,8 +8,14 @@ Create jira tickets for both the Bolt team and RE team for release. We use an in
 
 1. Clone [Winston](https://github.com/puppetlabs/winston) and `cd` into the directory 
 1. Run `bundle install` 
+1. If you've already cloned Winston, then **update from puppet master**. There's been some activity around Winston lately so make sure your local version is up to date!
```

Comment:
What does `update from puppet master` mean? That can be read several ways. I think it'd be clearer to say `update from puppetlabs/winston#master`.

Reply:
Agreed -- I wasn't really sure how to phrase it. Thanks!